marlowe-cli/src/Language/Marlowe/CLI/Run.hs

```diff
@@ -236,7 +236,9 @@ makeMarlowe marloweIn@MarloweTransaction{..} transactionInput@TransactionInput{.
                                    }
                                  )
                                    where
-                                     convertSlot = SlotNo . fromIntegral . getSlot
+                                     slotConfig = def
```

Comment:
The `def` instance is broken for `Ledger.TimeSlot.SlotConfig`, especially on private networks. This information will need to be pulled from command-line parameters or from `MarloweTransaction`.

Reply:
I'll fix it.